modules/cnv-define-readiness-probe.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc
+
+[id="cnv-define-readiness-probe_{context}"]
+
+= Define a readiness probe
+
+This procedure provides an example configuration file for defining
+Readiness probes.
+
+.Procedure
+
+. Customize a YAML configuration file to create a readiness probe. Readiness
+probes are configured in a similar manner to liveness probes. However, note the
+following differences in this example:
+* Readiness probes are saved using a different spec name. For example, you
+create a readiness probe as `spec.readinessProbe` instead of as
+`spec.livenessProbe.<type-of-probe>`.
+* When creating a readiness probe, you optionally set a `failureThreshold` and a
```

Comment:
It might be better to include this info as callouts in the code block itself, but it might be okay as is.